Recreate blog post PR 1142 (we did not have access rights)

alicejw-aws opened the pull request:
Tried to pull down team member's PR, [1142](https://github.com/opensearch-project/project-website/pull/1142) but access rights were denied. 

New PR created by copying file contents and images.

Signed-off-by: Alice Williams <alicejw@amazon.com>

### Description



### Check List
- [ x] Commits are signed per the DCO using --signoff


By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

alicejw-aws commented:
Hi @krisfreedain, here is the blog post we had to recreate it, as we didn't have access rights to update it directly. Could you please approve, it's got your comments updated from the original PR. 
https://github.com/opensearch-project/project-website/pull/1142
thanks

krisfreedain commented on _authors/hdhalter.markdown:
```diff
@@ -0,0 +1,8 @@
+---
+name: Heather Halter
+short_name: hdhalter
+photo: '/assets/media/authors/hdhalter.jpg'
```
looks like the file uploaded is .jpeg but this references .jpg

alicejw-aws replied:
so sorry. fixing now!

krisfreedain commented on _posts/2022-11-08-thudtest.md:
```diff
@@ -0,0 +1,72 @@
+In the years that followed, technical writers continued to face tough choices. We know that we have to write for customers first. We know that writing simply is hard. We know that we need to insist on the highest standards. We know that words are expensive to translate and resource intensive to change as the user interface changes.
+
+And we know that product managers still want to use the thud test to measure documentation output. <p style="max-width: 50%; float: right;">
+![Stack of papers](../assets/media/blog-images/stack%20of%20papers.jpg)</p>
```
looks like the file uploaded is .jpeg but this references .jpg

also, the path to your images needs to be resolved. See this example from another blog post. 

Example from another post: 
<img src="/assets/media/blog-images/2022-10-31-adding-new-distributions-in-opensearch-project/jenkins_pipeline_flowchart.png" alt="Jenkins Pipeliens Flowchart"/>{: .img-fluid }

you'll need to resolve all the image locations, and redo your path within each image line through this blog post.

alicejw-aws replied:
ok thanks!

krisfreedain reviewed (changes requested):
multiple issues with images - please resolve and we'll push this out - thank you

alicejw-aws commented:
hi @krisfreedain , i've updated the image paths and moved to similar directory with name of blog like the example you shared. please let me know if this looks correct now. 
thanks so much for your help!

krisfreedain reviewed (approved):
that last edit fixed it @alicejw-aws - thanks
